samples/nrf9160/aws_fota/README.rst

```diff
@@ -21,6 +21,8 @@ Requirements
   * nRF9160 DK board (PCA10090)
 
 * .. include:: /includes/spm.txt
+* AWS IoT account
+* Certificates with correct permissions to the AWS IoT MQTT broker written/flashed into the modem
```

Comment:
Is there any docs on how to do this?

Reply:
Tried to add some documentation on it below in the prerequisite section